<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointOptions</class>
 <widget class="QWidget" name="PointOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="sizeLabel">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="size">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
     <property name="pageStep">
      <number>4</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
     <item row="0" column="0">
      <widget class="QRadioButton" name="solidColor">
       <property name="text">
        <string>Solid Color</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ActorColorButton" name="color">
       <property name="color" stdset="0">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="trueColor">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>True Color</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="dataColor">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Color by Data</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="dataColorMenu">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>16</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="dataLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="dataField">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="dataFilter">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="dataFilterMenu">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="showVisibleLandmarks">
       <property name="text">
        <string>Show visible landmarks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="visibleLandmarksMenu">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="showVisibleLandmarksOnly">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Show visible landmarks only</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qtColorButton</class>
   <extends>QPushButton</extends>
   <header>qtColorButton.h</header>
  </customwidget>
  <customwidget>
   <class>ActorColorButton</class>
   <extends>qtColorButton</extends>
   <header>ActorColorButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>solidColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>dataColorMenu</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
